__tests__/errorReporting.spec.js

```diff
@@ -488,15 +489,20 @@ describe('error reporting', () => {
   describe('serverSideError', () => {
     it('should log the error', () => {
       expect.assertions(1);
-      const err = new Error('this is a test');
-      return serverSideError(err).then(() => {
-        expect(consoleErrorSpy).toHaveBeenCalledWith(util.inspect(err, false, 10, true));
+      const queue = [{
+        type: 'ServerSideReportedError',
```

Comment:
this should probably be an instance of `Error`? this property would be name instead of type, I think

Reply:
Removed it, not changing anything on the error now, the name is now just `Error`